packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item-utils.spec.ts

```diff
@@ -10,3 +10,12 @@ describe('getButtonAttributes()', () => {
     expect(getButtonAttributes(...args)).toMatchSnapshot();
   });
 });
+
+describe('getIconColor()', () => {
+  it.each<Parameters<typeof getIconColor>>([[false], [true]])(
+    'should return correct icon color for isDisabled: %s',
+    (...args) => {
+      expect(getIconColor(...args)).toMatchSnapshot();
```

Comment:
Could have been accomplished way easier to read and maintain without using snapshots.

Reply:
fixed it